lib/go/test/flow_lockedtokens_test.go

```diff
@@ -627,6 +627,27 @@ func TestLockedTokensStaker(t *testing.T) {
 		result = executeScriptAndCheck(t, b, templates.GenerateGetUnlockLimitScript(env), [][]byte{jsoncdc.MustEncode(cadence.Address(joshAddress))})
 		assertEqual(t, CadenceUFix64("8500.0"), result)
 	})
+
+	t.Run("Should be able to claim leased tokens as the admin", func(t *testing.T) {
+
+		script := templates.GenerateRecoverLeaseTokensScript(env)
+
+		tx := createTxWithTemplateAndAuthorizer(b, script, joshSharedAddress)
+
+		_ = tx.AddArgument(CadenceUFix64("10.0"))
+		_ = tx.AddArgument(cadence.NewAddress(adminAddress))
+
+		signAndSubmit(
+			t, b, tx,
+			[]flow.Address{joshSharedAddress},
+			[]crypto.Signer{adminSigner},
+			false,
+		)
+
+		// Check balance of locked account
+		result := executeScriptAndCheck(t, b, templates.GenerateGetFlowBalanceScript(env), [][]byte{jsoncdc.MustEncode(cadence.Address(joshSharedAddress))})
```

Comment:
shouldn't this be checking that the admin account is +10 flow?

Also, if it's easy, it'd be nice to check that the Unstaked bucket is -10 flow here as well

Reply:
yes, I'll add that

Reply:
Added the balance check. I can't check the unstaked bucket because we're just using a dummy contract to test here, but it should be fine